src/cable/handler.cr

```diff
@@ -36,17 +44,28 @@ module Cable
         socket.on_message do |message|
           begin
             connection.receive(message)
+          rescue e : KeyError
+            # handle unknown/malformed messages
+            socket.close(HTTP::WebSocket::CloseCode::InvalidFramePayloadData, "Invalid message")
+            Cable::Logger.error { "KeyError Exception: #{e.message}" }
           rescue e : Cable::Connection::UnathorizedConnectionException
             # do nothing, this is planned
+            socket.close(HTTP::WebSocket::CloseCode::NormalClosure, "Farewell")
+          rescue e : IO::Error
+            Cable::Logger.error { "#{e.class.name} Exception: #{e.message} -> #{self.class.name}#call { socket.on_message(message) }" }
+            # Redis may have some error, restart Cable
+            socket.close(HTTP::WebSocket::CloseCode::NormalClosure, "Farewell")
```

Comment:
In #54 @mjeffrey18 added the `socket.close`, but this rescue branch doesn't exist on master.... Should we be closing the socket here too? If so, then which CloseCode should be used?